src/cdk-experimental/ui-patterns/listbox/listbox.ts

```diff
@@ -270,6 +286,9 @@ export class ListboxPattern<V> {
     if (opts?.selectFromActive) {
       this.selection.selectFromActive();
     }
+    if (opts?.toggleFromAnchor) {
+      this.selection.toggleFromPrevSelectedItem();
```

Comment:
I think this should be `toggleFromFocusedItem`.

If I click an item to select, then press arrow down several times, then shift click, I'd expect selection to be starting from the focused item, not previously selected item.

Reply:
I don't think that's correct. Gmail & [React Aria](https://react-spectrum.adobe.com/react-aria/ListBox.html#selection) both select from the previous selected

Reply:
We prob shouldn't use gmail as a reference, though